jest.config.js

```diff
@@ -1,12 +1,22 @@
+/*
+ * For a detailed explanation regarding each configuration property, visit:
+ * https://jestjs.io/docs/configuration
+ */
+
 module.exports = {
-  coverageDirectory: './coverage',
-  coverageReporters: ['html', ['text', { skipFull: true }], 'text-summary'],
-  testEnvironment: 'node',
+  // Automatically clear mock calls, instances, contexts and results before every test
+  clearMocks: true,
+  // Indicates whether the coverage information should be collected while executing the test
+  collectCoverage: true,
+  // The directory where Jest should output its coverage files
+  coverageDirectory: 'coverage',
+  // Indicates which provider should be used to instrument code for coverage
   coverageProvider: 'v8',
-  moduleFileExtensions: ['js', 'json', 'ts'],
-  testRegex: ['.test.ts$'],
-  transform: {
```

Comment:
if you have switched to `ts-jest` you can remove `swc` from the dependencies

Reply:
Done that. Thanks